.github/scripts/update-releases.py

```diff
@@ -194,6 +281,25 @@ def process_authors(theme_designers, plugin_devs, overwrite=False, verbose=False
     print_file_summary(file_groups)
 
 
+def update_download_counts(verbose=True):
+    update_theme_download_counts(verbose)
```

Comment:
This is intended to leave space in case we add download counts to plugins, for example